openapi/paths/webhooks/webhooks@{webhookId}@dispatches.yaml

```diff
@@ -30,7 +30,7 @@ get:
         application/json:
           schema:
             allOf:
-              - $ref: ../components/schemas/Getcollectionresponse.yaml
```

Comment:
Can we reuse `GetListOfWebhookDispatchesResponse`?

Reply:
Yes, reused 👍